vocabularies/georesources-report-types.ttl

```diff
@@ -421,6 +461,24 @@ grt:petroleum-report-resource-and-reserves-information a skos:Concept ;
     skos:notation "RESINF" ;
     skos:prefLabel "Petroleum Report - Resource and Reserves Information"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/georesource-report> .
+
+grt:petroleum-report-surrender a skos:Concept ;
+    skos:altLabel "Final Surrender (ATP)"@en,
+        "Final Surrender (PL)"@en,
+        "Petroleum Report - Final Surrender"@en,
+        "Petroleum Report - Surrender"@en ;
+    skos:definition "A report that is required to be lodged with a partial of full surrender of a Petroleum Lease or Authority to Prospect, detailing the work undertaken on the permit throughout the life of tenure."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/georesource-report> ;
+    skos:notation "PETSUR" ;
+    skos:prefLabel "Petroleum Report - Surrender (ATP/PL)"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/georesource-report> .
+
+grt:pipeline-report-surrender a skos:Concept ;
+    skos:definition "A report that is required to be lodged with a partial of full surrender of a Petroleum Pipeline Lease, detailing the location of the pipeline and maintanence work undertaken throughout the life of tenure."@en ;

```

Comment:
partial of full surrender - think this is meant to be "or"

Reply:
```suggestion
    skos:definition "A report that is required to be lodged with a partial or full surrender of a Petroleum Pipeline Lease, detailing the location of the pipeline and maintanence work undertaken throughout the life of tenure."@en ;

```